packages/suite/src/actions/wallet/send/sendFormActions.ts

```diff
@@ -1,38 +1,61 @@
+// import * as storageActions from '@suite-actions/storageActions';
 import { Dispatch, GetState } from '@suite-types';
 import { SEND } from '@wallet-actions/constants';
+import { ETH_DEFAULT_GAS_LIMIT, ETH_DEFAULT_GAS_PRICE } from '@wallet-constants/sendForm';
 import { Account } from '@wallet-types';
 import { FeeLevel, Output } from '@wallet-types/sendForm';
-import { formatNetworkAmount, getAccountKey, getFiatValue } from '@wallet-utils/accountUtils';
+import { formatNetworkAmount, getFiatValue } from '@wallet-utils/accountUtils';
+// import { formatNetworkAmount, getAccountKey, getFiatValue } from '@wallet-utils/accountUtils';
 import { ParsedURI } from '@wallet-utils/cryptoUriParser';
 import { getOutput, hasDecimals, shouldComposeBy } from '@wallet-utils/sendFormUtils';
 import { getLocalCurrency } from '@wallet-utils/settingsUtils';
 import BigNumber from 'bignumber.js';
+// @ts-ignore
+import ethUnits from 'ethereumjs-units';
 
-import * as storageActions from '@suite-actions/storageActions';
-import * as bitcoinActions from './sendFormSpecific/bitcoinActions';
-// import * as ethereumActions from './sendFormSpecific/ethereumActions';
-import * as rippleActions from './sendFormSpecific/rippleActions';
+import * as bitcoinActions from './sendFormBitcoinActions';
+import * as ethereumActions from './sendFormEthereumActions';
+import * as rippleActions from './sendFormRippleActions';
+import * as commonActions from './sendFormCommonActions';
 
 /**
  * Initialize current form, load values from session storage
  */
 export const init = () => async (dispatch: Dispatch, getState: GetState) => {
-    const { settings, selectedAccount } = getState().wallet;
-    if (selectedAccount.status !== 'loaded') return;
-    const { account } = selectedAccount;
+    // const { settings, send } = getState().wallet;
+    const { settings } = getState().wallet;
+    const { account } = getState().wallet.selectedAccount;
+    if (!account) return;
 
-    let cachedState = null;
+    // let cachedState = null;
+    const convertedEthLevels: FeeLevel[] = [];
     const feeInfo = getState().wallet.fees[account.symbol];
-    const levels: FeeLevel[] = feeInfo.levels.concat({
-        label: 'custom',
-        feePerUnit: '0',
-        value: '0',
-        blocks: 0,
-    });
 
-    const accountKey = getAccountKey(account.descriptor, account.symbol, account.deviceState);
-    const cachedItem = await storageActions.loadSendForm(accountKey);
-    cachedState = cachedItem;
+    const initialLevels: FeeLevel[] =
+        account.networkType === 'ethereum'
+            ? feeInfo.levels
+            : feeInfo.levels.concat({
+                  label: 'custom',
+                  feePerUnit: '0',
+                  value: '0',
```

Comment:
you shoudnt use "value", it is not declared in FeeInfo and it's a deprecated filed from trezor-connect, use feePerUnit instead